### PR TITLE
Fix _version.py not being copied to /usr/lib/Auth/Facerec issue

### PR DESCRIPTION
**Pull request by morellil**

### CODE REVIEW

File: debian/install [hunk 1]
```diff
@@ -6,4 +6,4 @@ Facerec usr/lib/Auth/Facerec/
 config.py usr/lib/Auth/Facerec/
 src/cli/facerec usr/share/bash-completion/completions/
 pam_python.so lib/x86_64-linux-gnu/security/
-
+_version.py usr/lib/Auth/Facerec
```
Comment (rushabh-v) [inline debian/install]: Thanks for the PR. Please add a `/` at the end and put this on line 7 instead (with other files being copied to the same dir)
__Edit: ignore this!__

**Comment by rushabh-v** [comment]
resolves #22

File: debian/install (hunk 1 above)
Comment (rushabh-v) [reply debian/install]: Thinking more on this, I think moving _version.py to `./src/cli/` will be a better option as that script seems unnecessary in the home dir. Can you move that script under `./src/cli/` and remove this line (that you've added) from `debian/install`, please?

Comment (morellil) [reply debian/install]: Hi rushabh-v, changes done!

**Comment by rushabh-v** [comment]
Thanks, @morellil

**Pull request merged**